readme.md

```diff
@@ -3,6 +3,7 @@
 
 ## Table of Contents
 
+- [Table of Contents](#table-of-contents)
```

Comment:
Please remove unneeded link

Reply:
Done.

Reply:
Please un-lint the file and only add the resource.